lib/elixir/lib/enum.ex

```diff
@@ -2880,6 +2884,11 @@ defmodule Enum do
       iex> Enum.slice([1, 2, 3, 4, 5], 0..-1//2)
       [1, 3, 5]
 
+  To get every third element up to the 12th element:
```

Comment:
```suggestion
  To get every third element up to the 12th element:
```

If you want up to 12th, then you need 0..12 (or 0..11)?

Reply:
That's a bug!

```
iex(2)> Enum.slice(1..20, 0..10//2)
[1, 3, 5, 7, 9, 11]
iex(3)> Enum.slice(Enum.to_list(1..20), 0..10//2)
[1, 3, 5, 7, 9, 11, 13, 15, 17, 19]
```

The list version is wrong. :)

Also:

```
iex(4)> String.slice("abcdefghijklmnop", 0..10//2)
"acegik"
```

Reply:
Pushing a fix soon.

Reply:
Fixed in master.

Reply:
@josevalim nice, pushed a fix to this PR, let me know how it looks.